modules/cas_disk/exp_obj.c

```diff
@@ -652,6 +651,17 @@ struct request_queue *casdsk_exp_obj_get_queue(struct casdsk_disk *dsk)
 }
 EXPORT_SYMBOL(casdsk_exp_obj_get_queue);
 
+void casdsk_exp_obj_cleanup_queue(struct casdsk_disk *dsk)
+{
+	struct request_queue *exp_obj_q = NULL;
+
+	exp_obj_q = casdsk_exp_obj_get_queue(dsk);
+	BUG_ON(!exp_obj_q);
+
+	blk_cleanup_queue(exp_obj_q);
+}
+EXPORT_SYMBOL(casdsk_exp_obj_cleanup_queue);
```

Comment:
Why do we export this function? Is used only in this file, so it should be static.